timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/PaxosTimeLockServerIntegrationTest.java

```diff
@@ -238,6 +349,39 @@ public void throwsOnFastForwardWithIncorrectParameter() throws IOException {
         assertThat(response.code()).isEqualTo(HttpStatus.BAD_REQUEST_400);
     }
 
+    private List<Future<LockRefreshToken>> requestLocks(RemoteLockService lockService, int number, ExecutorService executorService) {
+        List<Future<LockRefreshToken>> futures = Lists.newArrayList();
+        for (int i = 0; i < number; i++) {
+            int currentTrial = i;
+            futures.add(executorService.submit(() -> lockService.lock(String.valueOf(currentTrial), SLOW_REQUEST)));
+        }
```

Comment:
heh, we can do `IntStream.range(0, number).boxed().map(i -> executorService.submit(() -> lock(...))).collect(Collectors.toList())` I think. Or I'm just being carried away with Java 8 stuff